fuse/ipns/ipns_unix.go

```diff
@@ -238,7 +239,11 @@ func (s *Node) Attr() fuse.Attr {
 			size = 0
 		}
 		if size == 0 {
-			size = s.dagMod.Size()
+			dmsize, err := s.dagMod.Size()
+			if err != nil {
+				log.Error(err)
```

Comment:
Attr is supposed to read the size of the object, If i have a pending write to the underlying dag, I choose to flush it and modify/append to the dag before getting the size (to make sure its accurate)